services/src/test/scala/cromwell/services/metadata/impl/DeleteMetadataActorSpec.scala

```diff
@@ -1,34 +1,69 @@
 package cromwell.services.metadata.impl
 
+import java.time.OffsetDateTime
+
 import java.util.UUID
 
 import akka.actor.Props
 import akka.testkit.{EventFilter, TestProbe}
 import cromwell.core.{TestKitSuite, WorkflowId}
-import cromwell.services.metadata.MetadataService.{DeleteMetadataAction, DeleteMetadataFailedResponse}
+import cromwell.services.metadata.impl.DeleteMetadataActor.DeleteMetadataAction
 import org.scalatest.FlatSpecLike
 
 import scala.concurrent.{ExecutionContext, Future}
 import scala.concurrent.duration._
+import scala.language.postfixOps
 
 class DeleteMetadataActorSpec extends TestKitSuite with FlatSpecLike {
 
-  private val workflowId = UUID.randomUUID().toString
-  private val deleteMetadataActor = system.actorOf(Props(new DeleteMetadataActor {
-    override def deleteNonLabelMetadataEntriesForWorkflow(rootWorkflowId: WorkflowId)(implicit ec: ExecutionContext): Future[Int] = {
-      Future.failed(new RuntimeException("Test exception"))
+  private val workflowId1 = UUID.randomUUID().toString
+  private val workflowId2 = UUID.randomUUID().toString
+
+  val probe = TestProbe()
```

Comment:
If you're only using this to send messages, you can use `ImplicitSender` instead to allow your test class to send messages itself:
```
class DeleteMetadataActorSpec extends TestKitSuite with FlatSpecLike with ImplicitSender
```